Cylenium.Tests/DriverTests.cs

```diff
@@ -39,6 +40,30 @@ public void Set_and_get_the_window_size()
             Assert.AreEqual(cy.WindowSize(), expectedSize);
         }
 
+        [Test]
+        [Category("Driver")]
+        public void Navigate_backward_and_forward()
+        {
+            var wait = cy.Wait();
+
+            cy.Visit("https://google.com");
+            wait.Until(_ => cy.Get("[name='q']"));
```

Comment:
`cy.Get()` already uses a Wait, so including it inside of `wait.Until()` is duplicating the Waits which will cause some weird side effects. Instead, just call `cy.Get()`.

```python
cy.Get("[name='q']");
```